doc/api/n-api.md

```diff
@@ -697,6 +697,58 @@ functions) that can be passed as parameters when `js_callback` is invoked. This
 pointer is managed entirely by the threads and this callback. Thus this callback
 should free the data.
 
+#### napi_key_collection_mode
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+todo
+
+```C
+typedef enum {
+  napi_key_include_all_properties,
```

Comment:
Would there be any possible ambiguity on these similar naming of `napi_key_include_all_properties ` and `napi_key_all_properties`? IIUC the `napi_key_collection_mode` indicates whether or not the keys on the prototype chains are collected, and `napi_key_filter` filters property descriptors.

Would it be possible to name `napi_key_collection_mode` following v8's pattern? i.e. `napi_key_include_prototypes`, which would sound very precise.

Reply:
I've changed them to `napi_key_include_prototypes` and `napi_key_include_own_properties` to follow v8's pattern, I think